ansible/setup.yml

```diff
@@ -24,6 +24,16 @@
       - sylane
     libsodium_version: "1.0.16"
     libsodium_module_version: "23.1.0"
+    fail2ban_loglevel: INFO
+    fail2ban_bantime: 86400
+    fail2ban_maxretry: 3
```

Comment:
Have you tested whether this 3 affects typical error of user attempting to SSH forgetting to load SSH key in agent in the first place? I am not sure if SSH CLI attempts multiple times under the hood, and how this behaves for a couple of users in same building.

(Distinct case of user's keypair change is expected to have a few bumps for at least a day.)

Reply:
Every invalid access attempt (password or key) will be detected. So using a wrong key and switching to the correct one, will be treated as one failed attempt. Multiple users behind the same IP address will all have 3 attempts for the same IP address. 

Fail2ban has a configuration option for the time-frame (findtime) for those attempts, but it is not configurable with the current Ansible role. The default (in the distribution provided global config) is 600 seconds. We could increase the retries to work-around those issues.